File: VB/13_Conversion/ToPPTX.vb
Imports Spire.Pdf

Namespace ToPPTX
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Specify the file path of the PDF document to be loaded
			Dim file As String = "..\..\..\..\..\..\Data\ToDocx.pdf"

			' Create a new PdfDocument object
			Dim doc As New PdfDocument()

			' Load the PDF document from the specified file path
			doc.LoadFromFile(file)

			' Convert the PDF document to a PPTX file format
			doc.SaveToFile("ToPPTX.pptx", FileFormat.PPTX)

			' Close the PDF document
			doc.Close()

			' Launch the file
			Process.Start("ToPPTX.pptx")
		End Sub
	End Class
End Namespace

File: CS/13_Conversion/ToPPTX.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Pdf;

namespace ToPPTX
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Load a pdf document
            String file = @"..\..\..\..\..\..\Data\ToDocx.pdf";
            PdfDocument doc = new PdfDocument();
            doc.LoadFromFile(file);

            //Convert to pptx file.
            doc.SaveToFile("ToPPTX.pptx", FileFormat.PPTX);
            doc.Close();

            //Launch the file.
            System.Diagnostics.Process.Start("ToPPTX.pptx");
        }
    }
}
